navigation/index.tsx

```diff
@@ -3,105 +3,43 @@
  * https://reactnavigation.org/docs/getting-started
  *
  */
-import { FontAwesome } from '@expo/vector-icons';
-import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
-import { NavigationContainer, DefaultTheme, DarkTheme } from '@react-navigation/native';
-import { createNativeStackNavigator } from '@react-navigation/native-stack';
-import * as React from 'react';
-import { ColorSchemeName, Pressable } from 'react-native';
-
-import Colors from '../constants/Colors';
-import useColorScheme from '../hooks/useColorScheme';
-import ModalScreen from '../screens/ModalScreen';
-import NotFoundScreen from '../screens/NotFoundScreen';
-import TabOneScreen from '../screens/TabOneScreen';
-import TabTwoScreen from '../screens/TabTwoScreen';
-import { RootStackParamList, RootTabParamList, RootTabScreenProps } from '../types';
-import LinkingConfiguration from './LinkingConfiguration';
-
-export default function Navigation({ colorScheme }: { colorScheme: ColorSchemeName }) {
+import {
+  NavigationContainer,
+  DefaultTheme,
+  DarkTheme,
+} from "@react-navigation/native";
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
+import { AuthScreenStack } from "../screens/AuthStack";
+import { HomeScreenStack } from "../screens/HomeStack";
+import * as React from "react";
+import { useState, useEffect } from "react";
+import { ColorSchemeName, Pressable } from "react-native";
+import NotFoundScreen from "../screens/NotFoundScreen";
+import { supabase } from "../supabase-service";
+import { RootStackParamList } from "../types";
+import LinkingConfiguration from "./LinkingConfiguration";
+
+export default function Navigation({
+  colorScheme,
+}: {
+  colorScheme: ColorSchemeName;
+}) {
+  console.log(supabase.auth.session());
+  const [auth, setAuth] = useState(false);
```

Comment:
Perhaps we could make this variable more meaningful:
```suggestion
  const [isAuthenticated, setIsAuthenticated] = useState(false);
```